substrate/frame/nis/src/tests.rs

```diff
@@ -17,21 +17,13 @@
 
 //! Tests for NIS pallet.
 
-use super::*;
-use crate::{mock::*, Error};
-use frame_support::{
-	assert_noop, assert_ok,
-	traits::{
-		fungible::{hold::Inspect as InspectHold, Inspect as FunInspect, Mutate as FunMutate},
-		nonfungible::{Inspect, Transfer},
-		tokens::{Fortitude::Force, Precision::Exact, Preservation::Expendable},
-	},
-};
-use sp_arithmetic::Perquintill;
-use sp_runtime::{
-	Saturating,
-	TokenError::{self, FundsUnavailable},
+use frame::{runtime::prelude::*, testing_prelude::*};
```

Comment:
```suggestion
use frame::testing_prelude::*;
```

The testing prelude already contains `runtime::prelude::*`